redpanda/utils/utils.go

```diff
@@ -251,6 +261,10 @@ func UserMechanismToString(m *dataplanev1alpha1.SASLMechanism) string {
 func TopicConfigurationToMap(cfg []*dataplanev1alpha1.Topic_Configuration) (types.Map, error) {
 	configs := make(map[string]attr.Value, len(cfg))
 	for _, v := range cfg {
+		if v.Value == nil {
+			// TODO should we skip, error, or set to empty string? skipping for now
```

Comment:
What is needed to complete this TODO?

Reply:
Honestly a conversation with you 😅

Reply:
Just checked back, this is used both in Read and Create. We should error here.

**In Create:**
An Empty value should not be accepted. Maybe the API could accept nil values and set defaults, but it's not great for us to save in the state an empty value and later in Read get a different value.

**In Read:**
Same, we shouldn't receive an empty value. 

An empty value is only needed for delete operations.